docs/processing/qaqc-criteria-FC.md

```diff
@@ -0,0 +1,63 @@
+# QA criteria for denoised data
+
+## FC distributions
+
+- [ ] Check that the FC distributions appear similar across sessions and are Gaussian centered approximately around zero with a small positive mean <sup>[1]</sup>. Two deviations from this scenario are possible:
+    1. The FC distributions of the majority of the sessions appear shifted towards positive values, skewed, flat, or bimodal. 
+    It indicates the denoising was either too aggressive for the number of functional frames available, or the BOLD signal is still contaminated with residual noise.
+    As such, the denoising needs to be revised as follows:
+        - [ ] Decrease the cut-off frequency of the low-pass filtering using the `--low-pass` flag of `funconn.py`
+        - [ ] If that modification is not enough, remove high-pass filtering by removing "high-pass" from the denoising strategy list. Do so by precising the flag `--confounds "motion" "scrubs"` to `funconn.py`.
+        - [ ] If those modifications are still not enough, decrease the cut-off frequency until the FC distributions of the majority of the subject is Gaussian centered approximately around zero with a small positive mean.
+
+        ??? note "Another tweak possible would be to increase the motion censoring threshold"
+            However, in our study we pre-registered that we will perform the analysis with three different motion censoring threshold.
+            As such, the analysis with high motion censoring threshold will be reported anyway.
```

Comment:
```suggestion
            Therefore, rather than optimizing the value of the motion censoring threshold, we will perform a *mulitverse* analysis and report the results for the three threshold levels we pre-registered.
```